diff --git a/src/css/quasar.variables.sass b/src/css/quasar.variables.sass
index cf59b8e..702f029 100644
--- a/src/css/quasar.variables.sass
+++ b/src/css/quasar.variables.sass
@@ -12,13 +12,14 @@
 // to match your app's branding.
 // Tip: Use the "Theme Builder" on Quasar's documentation website.
 
-$primary   : #1976D2
-$secondary : #26A69A
-$accent    : #9C27B0
+// Dracula Theme — https://draculatheme.com/spec
+$primary   : #BD93F9
+$secondary : #8BE9FD
+$accent    : #FF79C6
 
-$dark      : #1D1D1D
+$dark      : #282A36
 
-$positive  : #21BA45
-$negative  : #C10015
-$info      : #31CCEC
-$warning   : #F2C037
+$positive  : #50FA7B
+$negative  : #FF5555
+$info      : #8BE9FD
+$warning   : #FFB86C
